docs/advanced/Working_with_git.rst

```diff
@@ -0,0 +1,151 @@
++==================================+=====================================================+
+| git **clone**  <repo>            | Used to clone the repo ``git clone <repo> | <name>``|
++----------------------------------+-----------------------------------------------------+
+| git **commit**                   |  **Commit** an applied change on the given branch   |
++----------------------------------+-----------------------------------------------------+
+| git **remote**                   | `track <http://bit.ly/_fo_rk>`_ remote branch       |
++----------------------------------+-----------------------------------------------------+
+| ``git revert <commit-SHA-1>``    |`Revert <http://bit.ly/1FC01Bn>`_ changes in a commit|
++----------------------------------+-----------------------------------------------------+
+|``git fetch <remote> <branch>``   | The git `fetch <http://bit.ly/1xEtV2t>`_ command    |
+|                                  | imports commits or `tags <http://bit.ly/1ByA6X2>`_  |
+|                                  | from a remote repository into your local repo.      |
++----------------------------------+-----------------------------------------------------+
+| `git                             | **Updates** your repo. Shorthand for ``git fetch``  |
+| pull <http://bit.ly/_98as>`_     | followed by ``git merge FETCH_HEAD``. Recommended   |
+|                                  | to use it with `- -rebase <http://bit.ly/1yAi5bf>`_.|
++----------------------------------+-----------------------------------------------------+
+| **git log**                      | Lists commits made in the current **branch** of the |
+|                                  | repo. **Hacks**: ``git log --pretty=oneline``.      |
+|                                  | Also check `this <http://bit.ly/1zTFPCw>`_.         |
++----------------------------------+-----------------------------------------------------+
+| **git rebase** :                 |  - Rebasing is the process of moving a **branch**   |
+|  - ``git rebase <base>``         |    to a new **base commit**                         |
+|  - ``git rebase -i <base>``      |  - Interactive `rebase <http://bit.ly/1w0XRkd>`_    |
+|  - ``git rebase -i HEAD~NUM``    |  - Modifying to a `head <http://bit.ly/1As3Raw>`_   |
+|  - ``git rebase -i bbc643cd^``   |  - Modify to specified commit bbc643cd              |
+|  - ``git rebase --abort``        |  - Abort a rebase                                   |
+|  - ``git reflog``                |  - Tracks the changesets to the tip of branch       |
++----------------------------------+-----------------------------------------------------+
+
+Issues with pull Requests
+=========================
+
+Problems new contributors face
+-------------------------------
+
+1. **Understand** your `problem <https://sethrobertson.github.io/GitFixUm/fixup.how-can-i-switch-my-git-repository-to-a-particular-commit>`_ first.
```

Comment:
In general, we try to keep files to 80-character line width if possible. I know this can be difficult or impossible with long links, but can you give that a try for the lines with shorter links?